smart_campus/app/templates/app/login.html

```diff
@@ -11,44 +11,53 @@
 {% block nav %}{% endblock %}
 
 {% block content %}
-<div class="container">  
-  <div class="col-lg-6 col-md-6 col-sm-8 loginbox">
-    <div class="row">
-      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-6">
-        <img src="{% static 'app/icon/ncku.png' %}" alt="Logo" class="logo"> 
+<div class="ui middle aligned center aligned grid">
+  <div class="column">
+    <h2 class="ui teal header">
+      <!-- <img src="{% static 'app/icon/ncku.png' %}" class="logo"> -->
```

Comment:
**DO NOT** version control **ANY** code that is commented out.

Reply:
okay, removed it.